### ISSUE TITLE
travis: shogun-python-modular not available

### ISSUE THREAD

**Issue by nno** [PyMVPA/PyMVPA#540]
Some recent build failed. e.g.:

https://travis-ci.org/PyMVPA/PyMVPA/jobs/281753402

```
Package shogun-python-modular is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source
```

**Comment by nno** [comment]
Some attempt to fix this was started by @yarikoptic in #534

**Comment by yarikoptic** [comment]
#534 was merged, we seems to be in greens so closing
